### PR TITLE
Cosmos DB Spark: Making sure users can obtain _ts as Timestamp

### LINKED ISSUE #20224: Discuss whether to use long or spark timestamp for _ts.​

### PR DESCRIPTION
**Pull request by ealsur**
`_ts` is a field that is `Long` by default, but users can use a custom schema and obtain it as Timestamp with `StructField("_ts", TimestampType)`

Closes #20224

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
Thanks

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EQueryITest.scala [hunk 1]
```diff
@@ -361,6 +364,56 @@ class SparkE2EQueryITest extends IntegrationSpec with Spark with CosmosClient wi
...
+    val rowsArray = df.collect()
+
+    for (index <- 0 until rowsArray.length) {
+      val row = rowsArray(index)
+      val ts = row.getAs[Timestamp]("_ts")
+      val id = row.getAs[String]("id")
+
+      ts.getTime() > 0 shouldBe true
+
+
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EQueryITest.scala:408]: nit: please cleanup empty lines.

**Review by moderakh — APPROVED** [review]
LGTM.

**Pull request merged**